dev-tools/ecs-migration.yml

```diff
@@ -44,3 +44,43 @@
   to: agent.version
   alias: true
   copy_to: false
+
+- from: source_ecs.ip
+  to: source.ip
+  alias: true
+  copy_to: false
+
+- from: source_ecs.port
+  to: source.port
+  alias: true
+  copy_to: false
+
+- from: source_ecs.geo.continent_name
```

Comment:
Needlessly verbose, IMO. What about the following?

```YAML
- from: source_ecs.geo
  to: source.geo
```

Reply:
I don't know how `ecs-migration.yml` will be used so I can't say for sure whether this is needlessly verbose. But if I can speculate, I think the level of detail here is appropriate if this will be used to create aliases because aliases [must target](https://www.elastic.co/guide/en/elasticsearch/reference/6.5/alias.html#alias-targets) concrete fields, not objects like `source.geo`.

Reply:
Good point